src/app/zap-templates/zcl/data-model/chip/commissioning.xml

```diff
@@ -107,17 +107,17 @@ limitations under the License.
             <description>TODO</description>
             <arg name="SSID" type="OCTET_STRING"/>
             <arg name="Credentials" type="OCTET_STRING"/>
-            <arg name="breadcrumb" type="INT64U"/>
+            <arg name="breadcrumb" type="INT64U" optional="true"/>
         </command>
         <command source="client" code="0x03" name="AddOrUpdateThreadNetwork" optional="true" response="NetworkConfigResponse" cli="chip network_commissioning addorupdatethreadnetwork">
             <description>TODO</description>
```

Comment:
Can you please fix all the TODO descriptions while you're in here?

Reply:
Done.